app/workers/content_block_publishing_worker.rb

```diff
@@ -0,0 +1,62 @@
+require "sidekiq/api"
+
+# ContentBlockPublishingWorker is a job that is scheduled by the `ScheduleEditionService`.
+# It is never executed immediately but uses the Sidekiq delay mechanism to
+# execute at the time the edition should be published.
+class ContentBlockPublishingWorker < WorkerBase
```

Comment:
perhaps this should be called `ContentBlockSchedulerWorker` or have `schedule` in it? 🤔